environments/staging/known_hosts

```diff
@@ -55,6 +55,9 @@
 10.201.41.168 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBJtIyQ/hxk75Am6F2JkgrhpvpJB3WojtzdTl7HXFDTwIIiWgXgeswt6VU28V8QoQ73LbUS51qDYhgoo3JiSQ3aU=
 10.201.41.168 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIFrDaj4Gl5jW+clYIPvcIaLhj8LqoiuWYUVghUT+o8PO
 10.201.41.168 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDKKb1MqDmZl4tlXkPeV7a1JQaQDG40Ec1o3RJ/75qJZKzFQqxPTrSAWqvJbshjQPN/r2fb6RcXL8wN2BW0PAANUDrO6wzTZF63GMJhVQN9Nff9y4oSkXRJ9PU96koTGsY6mHiYLV8kzTNugBGAkHmK3+XtyeDFgQk3rKWUmPdhZ+Nt685Ue0sQrgs4YgIKNiZq5GB99yUz0q1hVhICvr46F/K6otpgwlZhgvUoIi5uMr+AYlz2Mpz5jGnCgSUHH8WU2/5tkft87pS4B1xXmMlrmgwix4XfDOXWmSz5loKtCLURHMf74kdpOkeQKgFc4lir+GknyXx3CIdmIxB0xyNXExzBA7NJPeqfLf6BkY307z8lg9+9GM/S+fpOPw+klDyzF4WU8OJBqlIWuyPftxQkX7ma5fYYf0wso6hXcPho2qcEX/l+tW5S4+G53OTezD/3cwMYXP79xO9eVi3wbMoGiX95RRkAQ18AahSmpS4OLvUwuniOoTD97L+l5EB3Suc=
+10.201.41.18 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBEJ3vAfvGwCxk0j8YLSsHGAZpk6TDHdb6LSHmHveVJ4NaEe6oxTaud9q6b/oOqPAAoPeHaT5/8Jn/Fvi+Vz939Q=
```

Comment:
is it worthwhile to remove 10.201.40.86 from this file. It should not do any harm. I am just wondering if it would be cleaner.